apps/web/components/sections/project/about-project.tsx

```
@@ -0,0 +1,96 @@
+	titleAboveHighlights,
+}) => {
+	return (
+		<div className="about-project mt-6 bg-white rounded-lg p-6 shadow-md font-inter">
+			<div className="mb-6">
+				<h3 className="text-xl font-bold text-black mb-2">
+					About this project
+				</h3>
+				<p className="text-gray-600">{description}</p>
+			</div>
+
+			<div className="border-t border-gray-200 my-4"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)